homeassistant/components/zoneminder.py

```diff
@@ -79,46 +78,45 @@ def login():
         ZM['url'] + 'api/host/getVersion.json', cookies=ZM['cookies'],
         timeout=DEFAULT_TIMEOUT)
 
-    if req.status_code != requests.codes.ok:
+    if not req.ok:
         _LOGGER.error("Connection error logging into ZoneMinder")
         return False
 
     return True
 
 
-# pylint: disable=no-member
-def get_state(api_url):
-    """Get a state from the ZoneMinder API service."""
+def _zm_request(method, api_url, data=None):
+    """Perform a Zoneminder request."""
+    import simplejson
```

Comment:
That's the actual exception raised by requests when `result.json()` encounters invalid JASON data. But yeah `ValueError` should do the same job. I'll update this later.